src/components/Board.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <div v-if="gameStats.gameStart">
     <h2>{{ msg }}</h2>
-    
+    <div class="wrapper">
+      <div v-for="(row, index) in gameSquares" :key="index" class="row">
+        <p>{{ index }}</p>
```

Comment:
here I am just printing out the `index` value into the page so you can see the row names (such as `row8`, `row7`, etc).